src/java/main/org/apache/zookeeper/client/ConnectStringParser.java

```diff
@@ -20,8 +20,11 @@
 
 import java.net.InetSocketAddress;
 import java.util.ArrayList;
+import java.util.List;
 
 import org.apache.zookeeper.common.PathUtils;
+import static org.apache.zookeeper.common.StringUtils.split;
```

Comment:
This code is similar to existing one in master and branch-3.5. For consistency, we could use what kind of import it uses, imo.